packages/sdk/src/evm/functions/getContractFromAbi.ts

```diff
@@ -26,13 +26,14 @@ export type GetContractFromAbiParams = {
 export async function getContractFromAbi(
   params: GetContractFromAbiParams,
 ): Promise<SmartContract> {
-  const resolvedAddress = await resolveAddress(params.address);
-
   const [signer, provider] = getSignerAndProvider(
     params.network,
     params.sdkOptions,
   );
-  const chainId = (await provider.getNetwork()).chainId;
+  const [resolvedAddress, {chainId}] = await Promise.all([
```

Comment:
make sure your prettier config is setup correctly

Reply:
Thanks! I reformat this part and include it in the next PR